core/src/utils/theme.ts

```diff
@@ -81,6 +81,12 @@ export const generateCSSVars = (theme: any, prefix: string = CSS_PROPS_PREFIX):
         return [];
       }
 
+      // Do not generate CSS variables for excluded keys
+      const excludedKeys = ['enabled', 'ripple-effect', 'form-highlight'];
+      if (excludedKeys.includes(key)) {
+        return [];
+      }
+
```

Comment:
Is it intended that this new code runs before the camelCase to kebab-case switch? This means these keys may exist in camelCase and not be excluded because they haven't been switched yet.